app/src/main/java/com/brandon/forzenbook/viewmodels/CreateAccountViewModel.kt

```diff
@@ -6,6 +6,7 @@ import androidx.compose.runtime.mutableStateOf
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import com.brandon.createaccount.core.usecase.CreateUserUseCase
+import com.brandon.logincompose.viewmodel.LoginViewModel
```

Comment:
Unused import?

Reply:
Going to investigate this, I saw it in one of the files and it wasn't greyed out, meaning it got use somewhere, likely for a log constant if I had to guess

Will remove it since I shouldn't be pulling in another viewmodel in there anyways

Reply:
Removed this import, it was bringing in a log tag from the other viewmodel. Added a log tag for this viewmodel in the companion object